src/index.html

```diff
@@ -11,25 +11,49 @@
   <body>
 
     <header class="header">
-      <div class="container">
-        <a href="index.html"> <img class="logo-img"
+
+      <a class="a" href="index.html"> <img class="logo-img"
           src="./images/logo.png" alt="logo">
-        </a>
-        <nav class="nav">
-          <ul class="ul-decor">
-            <li class="li-set"> <a class="is-active nav-set" href="#apple">APPLE</a></li>
-            <li class="li-set"> <a class="nav-set" href="#samsung">SAMSUNG</a></li>
-            <li class="li-set"> <a class="nav-set" href="#smart">SMARTPHONES</a></li>
-            <li class="li-set"> <a class="nav-set" data-qa="hover"
-              href="#lap&comp">LAPTOPS & COMPUTERS</a>
-            </li>
-            <li class="li-set"> <a class="nav-set" href="#gadg">GADGETS</a></li>
-            <li class="li-set"> <a class="nav-set" href="#tabl">TABLETS</a></li>
-            <li class="li-set"> <a class="nav-set" href="#photo">PHOTO</a></li>
-            <li class="li-set"> <a class="nav-set last" href="#video">VIDEO</a></li>
-          </ul>
-        </nav>
-      </div>
+      </a>
+
+      <nav class="nav">
+        <ul class="ul-decor">
+          <li class="li-set">
+            <a class="is-active nav-set a" href="#apple">apple</a>
+          </li>
+
+          <li class="li-set">
+            <a class="nav-set a" href="#samsung">samsung</a>
+          </li>
+
+          <li class="li-set">
+            <a class="nav-set a" href="#smartphones">smartphones</a>
+          </li>
+
+          <li class="li-set">
+            <a class="nav-set a" data-qa="hover"
+              href="#laptops&computers">laptops & computers
+            </a>
+          </li>
+
+          <li class="li-set">
+            <a class="nav-set a" href="#gadgets">gadgets</a>
+          </li>
+
+          <li class="li-set">
+            <a class="nav-set a" href="#tablets">tablets</a>
+          </li>
+
+          <li class="li-set">
+            <a class="nav-set a" href="#photo">photo</a>
+          </li>
+          
+          <li class="li-set">
+            <a class="nav-set last a" href="#video">video</a>
+          </li>
+        </ul>
+      </nav>
+
```

Comment:
Let's remove this empty line (same with 18 line of code)

![image](https://github.com/mate-academy/layout_moyo-header/assets/96138503/82dde994-b377-4148-aaa7-b3817e6d3e15)